packages/react-i18n/src/i18n.ts

```diff
@@ -150,6 +156,39 @@ export default class I18n {
     return this.formatNumber(amount, {as: 'currency', ...options});
   }
 
+  unformatCurrency(input: string, currencyCode: string): string {
+    const nonDigits = /\D/g;
```

Comment:
There is one problem I see with this: Japanese full-width characters/numbers. See: https://stackoverflow.com/questions/46290595/japanese-regex-for-numbers

But that's another issue, I definitely won't block this PR because of this. It's more of an FYI/TIL 👍